bus-mapping/src/evm/opcodes.rs

```diff
@@ -133,6 +136,16 @@ impl Opcode for Dummy {
     }
 }
 
+#[allow(unused_variables)]
+impl TxExecSteps for Dummy {
+    fn gen_associated_steps(
+        state: &mut CircuitInputStateRef,
+        execution_step: ExecState,
+    ) -> Result<ExecStep, Error> {
+        Ok(state.new_begin_tx_step())
```

Comment:
i think either `state.new_step(&geth_steps[0])` or even `unimplemented!("...")` would be better implementation for this. `state.new_step(&geth_steps[0])` seems to be good default implementation, while `unimplemented` at least it reports error. While, state.new_begin_tx_step() is a wrong implementation for most cases.

Reply:
just realized `impl TxExecSteps for Dummy` could be reomved and fixed in [1dcc3ac](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/1336/commits/1dcc3ac0bb483108471a70d2c054cdcdbff955ec)